provisioners/file/var/spool/cron/tabs/jenkins

```diff
@@ -0,0 +1,4 @@
+# DO NOT EDIT THIS FILE - edit the master and reinstall.
+# (C installed on Thu Apr  5 21:42:22 2018)
```

Comment:
Technically this _is_ the master file, since you're not using `crontab -e` here.

Reply:
Sure, will fix.

Reply:
Done with commit 02aa7bfb831474f63a5c0d24456a99b47906f446